packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -161,11 +162,19 @@
         feature: this.queryFeature,
         location: this.location
       });
-
+      const filtersApply = this.queryPreviewInfo.filters?.reduce((filtersList, filterId) => {
+        const facetId = filterId.split(':')[0];
+        const rawFilter = createRawFilter(filterId);
+        filtersList[facetId] = filtersList[facetId]
+          ? filtersList[facetId].concat(rawFilter)
+          : [rawFilter];
+        return filtersList;
```

Comment:
Add space here for clarity.

```suggestion

        return filtersList;
```